packages/react-server/src/ReactFlightServer.js

```diff
@@ -839,6 +839,7 @@ function emitModuleChunk(
   id: number,
   moduleMetaData: ModuleMetaData,
 ): void {
+  // $FlowFixMe ModuleMetaData is not a ReactModel
```

Comment:
Couldn't figure this one out how this is supposed to be allowed…

Reply:
I can follow up on this one.